src/commands/helpcard/helpcard.command.ts

```diff
@@ -0,0 +1,25 @@
+
+import * as path from 'path';
+
+import { Message, Attachment } from 'discord.js';
+import { Command, CommandClass, helpcardService } from '../../shared';
+import { prefixedCommandRuleTemplate } from '../../config';
+
+@Command({
+  matches: ['helpcard'],
+  ...prefixedCommandRuleTemplate
+})
+
+export class HelpcardCommand implements CommandClass {
+
+    action(msg: Message, args: string[]) {
+
+        if (args.length <= 1) msg.channel.send('Invalid command usage! You must specify a helpcard name: ``!helpcard "example_name_here"`` (without the quotes).');
+
+        if (helpcardService.imageNames.includes(args[1].replace(/\s+/g, '_'))) {
```

Comment:
Is the intent of the regex replacement to allow for spaces in a function name? If so, this wouldn't work how you expect since `.replace` only returns a copy, instead of editing in place, so if someone provided `draw sprite` this would cause an error on line 20, where you reference the un-replaced arg in the image (i.e. `../draw sprite.png` is incorrect).

I think this is proooobably not needed anyway, since referencing `draw_sprite` as `draw sprite` just feels wrong, and is also inconsistent with how `!docs` works, since that command doesn't allow that format as either.

If you want to include it you may, but then you'll have to change where you reference it for the attachment.

Other than that, everything else is good! Making a service for this was the right way to go 👍